src/test/java/no/sikt/nva/scrapers/DublinCoreScraperTest.java

```diff
@@ -1151,6 +1151,17 @@ var record = dcScraper.validateAndParseDublinCore(dublinCore, brageLocation, SOM
         assertThat(record.getPublication().getPublicationContext().getJournal().getPid(), is(equalTo(expectedPid)));
     }
 
+    @Test
+    void shouldReadPublisherPidForInstitutionIssuingDegree(){
+        var type = toDcType("Bachelor thesis");
+        var publisher = new DcValue(Element.PUBLISHER, null, "gurba");
+        var brageLocation = new BrageLocation(null);
+        var dublinCore = DublinCoreFactory.createDublinCoreWithDcValues(List.of(type, publisher));
+        var record = dcScraper.validateAndParseDublinCore(dublinCore, brageLocation, "ntnu");
+        var expectedPid = "D61B0D47-C78A-48DC-8537-3AD87DEF4D5B";
+        assertThat(record.getPublication().getPublicationContext().getPublisher().getPid(), is(equalTo(expectedPid)));
+    }
+
```

Comment:
Nice, but we need also some functionality in publication-api so we prioritize this publisher over publisher from existing publication, when both publishers are verified.

Reply:
*when merging with publication from Cristin